packages/xstate-graph/test/graph.test.ts

```diff
@@ -347,6 +347,39 @@ describe('@xstate/graph', () => {
 
       expect(adj).toHaveProperty('"full" | {"count":5}');
     });
+
+    it('should get events via function', () => {
```

Comment:
Something weird has happened here as well - this branch contains changes from the other PR: https://github.com/davidkpiano/xstate/pull/1663

I know it's somewhat nitpicky but such things truly make it harder to focus on the given PR, maybe not in here because the changes are nicely isolated to `@xstate/graph` but in general this can be truly problematic if changes affect the same package.

A side question: do you know how this has happened? 🤔 I'm curious from the technical PoV - the commit that introduces this (https://github.com/davidkpiano/xstate/commit/d2a59a92cce84dba5c212f654e30de2a6d85a9ea) is the same commit on all 3 branches but there are no merges between those branches as far as I can tell so how did it even get onto them as the same commit? Git is still a mystery to me sometimes 😅

Reply:
Oh, my hypothesis is that you have merged in the #1663 **locally** and merged the local version of `master` into 2 other branches. That would kinda explain it (maybe?).